### ISSUE TITLE
[bug] Exported identifiers do not match with imported ones

### ISSUE THREAD

**Issue by Andarist** [brigand/babel-plugin-flow-react-proptypes#164]
Importing file has this:
```js
import { bpfrp_babelPluginFlowReactPropTypes_proptype_CellRenderer } from './types'
```
where export looks like this:
```js
export { babelPluginFlowReactPropTypes_proptype_CellRendererParams };
```

**Comment by brigand** [comment]
@Andarist could you give an example that reproduces the issue?

Tried:

```js
// a.js
export type A = {}
```

```js
// b.js
import { type A } from './a.js';
```

Outputs

```js
// a.js after compile
var babelPluginFlowReactPropTypes_proptype_A = {};
export { babelPluginFlowReactPropTypes_proptype_A };
```

```js
// b.js after compile
import './a.js';
import { bpfrp_babelPluginFlowReactPropTypes_proptype_A } from './a.js';
```


Also this

```
// b.js
import type { A } from './a.js';

// b.js after compile
import { bpfrp_babelPluginFlowReactPropTypes_proptype_A } from './a.js';
```

**Comment by Andarist** [comment]
I have just came back home and its past 1 am, so cant do much right now, but your examples from what I see are also broken - notice how exported (`babelPluginFlowReactPropTypes_proptype_A`) specifier does not match to imported one (`bpfrp_babelPluginFlowReactPropTypes_proptype_A`)

**Comment by brigand** [comment]
Oh lol, missed that.

**Comment by brigand** [comment]
Published a fix as 14.0.2
